Package.swift

```diff
@@ -21,7 +21,7 @@ import PackageDescription
 let package = Package(
```

Comment:
We likely want to change the swift tools version to 6.0 at the top of the file, and remove the stray `let version` line at the bottom of this file